includes/library.php

```diff
@@ -428,6 +428,27 @@ function bu_navigation_get_posts( $args = '' ) {
 		);
 	$r = wp_parse_args( $args, $defaults );
 
+	// Get the `last_changed` key from the WP core `posts` cache group.
+	// This key is updated by core in `clean_post_cache`.
+	$last_changed = wp_cache_get( 'last_changed', 'posts' );
+
+	// If WP's `last_changed` key has no value, set it using the current time.
+	if ( ! $last_changed ) {
+		wp_cache_set( 'last_changed', microtime(), 'posts' );
```

Comment:
Actually, the above function would mean you wouldn't have to do this if im reading the source code correctly for `wp_cache_get_last_changed`

Reply:
That looks right to me, too! Thanks for the suggestion!